<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BG4NB0447" failures="1" tests="2" name="Test" time="27.797" errors="0" timestamp="2024-10-24T14:24:17 IST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="3.09"/>
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="10.857">
    <failure type="java.lang.AssertionError" message="login Passed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: login Passed expected [true] but found [false]
at testCases.TC002_LoginTest.verify_login(TC002_LoginTest.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
</testsuite> <!-- Test -->
